dnszone_add.go

```diff
@@ -0,0 +1,38 @@
+package bunny
+
+import "context"
+
+// DNSZoneAddOptions are the request parameters for the Get DNS Zone API endpoint.
+//
+// Bunny.net API docs: https://docs.bunny.net/reference/dnszonepublic_add
+type DNSZoneAddOptions struct {
+	ID int64 `json:"Id,omitempty"`
+
+	Domain                        string      `json:"Domain,omitempty"`
+	Records                       []DNSRecord `json:"Records,omitempty"`
+	DateModified                  string      `json:"DateModified,omitempty"` // should be time
+	DateCreated                   string      `json:"DateCreated,omitempty"`  // should be time
```

Comment:
please clarify the "should be time" comments and document in which format the timestamp is expected

Reply:
I've updated this and documented the time format in the comments. It seems like elsewhere in the codebase (e.g. `storagezone_get.go`) this isn't documented though; perhaps it would be worth moving timestamp docs to the README or elsewhere since it seems like Bunny is using the same timestamp format across their API?

Reply:
Yes, true.

Adding the `// Timestamp` comments that you introduced to all those fields and having a package level godoc that explains the format would also be a good way.